source/beamline/AMPVControl.cpp

```diff
@@ -92,6 +95,18 @@ void AMReadOnlyPVControl::onReadPVError(int errorCode) {
 	}
 }
 
+void AMReadOnlyPVControl::setMinimumValue(double newValue)
+{
+	setLowLimitValue(newValue);
```

Comment:
Should there be a check here if the manually set override minimum is less than any value which has come from epics? It seems like it might be a bad idea to allow acquaman to override a limit with a value which is less constraining than the one which comes from epics?

I mean, the motor still won't get there if we tell it to go beyond the limits, so it's not all that bad. The setpoint will get pushed to epics, which will simply stop it at the limit value. Maybe an AMWarning?